pkgs/development/compilers/eunomia-bpf/eunomia-cc/default.nix

```
@@ -0,0 +1,128 @@
+{ lib
+, stdenv
+, makeWrapper
+, fetchFromGitHub
+, rustPlatform
+, cmake
+, pkg-config
+, llvmPackages_16
+, cargo
+, rustc
+, elfutils
+, zlib
+, ...
+}:
+let
+  bpftool =
+    stdenv.mkDerivation {
```

Inconsistent with other indentation in the file, again I'd suggest:

```suggestion
  bpftool = stdenv.mkDerivation {
```